### PR TITLE
Various touchups to Travis Build

### PR DESCRIPTION
**Pull request by ccoffrin**
Turns out Jullia v0.6 was working, on linux at least...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,26 +4,31 @@ os:
...
+  - julia -e 'if !("Ipopt" in keys(Pkg.installed())) Pkg.add("Ipopt") end'
+  - julia -e 'if !("Coverage" in keys(Pkg.installed())) Pkg.add("Coverage") end'
+
 script:
- - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
- - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("AmplNLWriter"); Pkg.add("Ipopt"); Pkg.test("AmplNLWriter")'
-before_cache:
-  - cp -R $HOME/.julia/*/Ipopt/deps/usr $HOME
+  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("AmplNLWriter"); Pkg.test("AmplNLWriter")'
```
Comment (odow) [inline .travis.yml]: Can you change this to `Pkg.test("AmplNLWriter", coverage=true)` while you're at it? I forgot to do it when I added the .codecov.yml

**Comment by odow** [comment]
Not really sure why this failed on linux https://travis-ci.org/JuliaOpt/AmplNLWriter.jl/jobs/306473924 but not osx or windows. Should we just scrap the caching?

**Comment by odow** [comment]
Travis now failing (on linux only) as JuMP still contains reference to BonminNLSolver.
https://github.com/JuliaOpt/JuMP.jl/blob/master/test/solvers.jl#L148-L149

**Comment by mlubin** [comment]
There will be no more calling JuMP from solver wrapper tests under MOI.

**Pull request merged**